src/erl_optics_app.erl

```diff
@@ -19,6 +19,7 @@ start(_Type, _StartArgs) ->
 
 stop(_State) ->
     supervisor:terminate_child(erl_optics_sup, erl_optics_server),
+    exit(whereis(erl_optics_sup), kill),
```

Comment:
I think I was wrong about this. After reading a bit more, it looks to me like calling `application:stop` will kill the top supervisor. This means you don't need to do anything in this callback, so you should just return `ok`